addons/binding/org.openhab.binding.zwave/src/main/java/org/openhab/binding/zwave/internal/protocol/commandclass/ZWaveAlarmCommandClass.java

```diff
@@ -368,6 +417,8 @@ public String getLabel() {
         @XStreamOmitField
         boolean initialised = false;
 
+        List<Integer> reportedEvents = new ArrayList<>();
```

Comment:
I don't think we should access this directly - it would be preferable to use getter/setter methods as we do already for other properties in this class.